### DISCUSSION TITLE
Enough marketing! Where's the Maui demos?

### DISCUSSION THREAD

**Discussion by Joebeazelman** [dotnet/maui#222] [General] [answered]
It would be nice to have some prebuilt demos and samples we can download to see what this Maui hype is all about. Of course, that would be too sensible for Microsoft!  So I cloned the entire Maui repo and tried running the obvious script build.sh script, which failed. There was nothing in the repo on how to build it, so I spent another 15 minutes trying to locate instructions for how to build maui on this project site and found nothing. Nada. Zippo!  I did, however, find tons of marketing spiel and FAQ questions which still didn't address how to build Maui. 

Microsoft manages to botch even the simplest things! The only reason I put up with any of this is because of C#/.NET, but I am now seriously considering dumping the whole platform. Does it even make sense to trust Microsoft's judgement for building any kind of user interface, when it consistently fails at usability? It also doesn't help that Microsoft is now coddling UNIX and its developers who have no concept or care for usability or proper documentation. The entire toolchain is now polluted with user hostilite paradigms. Even though Apple is based on UNIX, they're human enough to give developers a pleasant user experience.

**Comment by vdvman1** [comment]
MAUI isn't available yet, it is still in the very early stages of development

**Comment by Happypig375** [comment]
It also doesn't help that you didn't bother to read and understand that MAUI will be released officially in November 2021 and development will not start until Xamarin.Forms 5.

Please use your eyes and brain instead of complaining.

**Reply by Joebeazelman** [reply]
Well pardon me, Mr. Brainiac! The huge volume of source code, along with the statement below from the readme file leads any NORMAL person to conclude that though it's still in its infancy, there's some executable samples.

```
Active development is happening today to build Android and iOS SDKs against the next version of .NET. Samples may be found here.

Goals

Improve app performance
Improve simplicity of control extensibility
Improve simplicity of contributing
Enable developer options to use Model-View-Update (MVU) and Blazor

```

A real brainiac wouldn't obsequiously defend a multibillion dollar corporation, especially if they're not paying you. You would be the first to sign up to have that brilliant brain of yours containerized and deployed to Microsoft's Singularity clusters.  I'm sure you'll feel right at home in your  lowpoly, user hostile and bug ridden Microsoft-based reality. Just hope it doesn't crash or boot loops for eternity. After all, a mind is a terrible thing to waste!

**Comment by PureWeen** [comment, marked as answer]
Previews are planned for early 2021. We can't even release previews until .NET 6 is out
